bundles/com.amazonaws.eclipse.core/src/com/amazonaws/eclipse/explorer/sqs/QueueEditor.java

```diff
@@ -163,7 +163,7 @@ public void run() {
 
             while (iterator.hasNext()) {
                 Message message = iterator.next();
-                getClient().deleteMessage(new DeleteMessageRequest(queueEditorInput.getQueueUrl(), message.getReceiptHandle()));
+                getClient().deleteMessageBatch(new DeleteMessageRequest(queueEditorInput.getQueueUrl(), message.getReceiptHandle()));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code makes a call to a batch operation, but it does not handle any errors returned by that batch operation. Batch operations can succeed without throwing exceptions even if some items in the request fail. Consider checking the return value of the batch operation for failures.